addons/bus/static/src/debug/bus_logs_menu_item.js

```diff
@@ -0,0 +1,39 @@
+import { Component, useRef, useState } from "@odoo/owl";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+/**
+ * @typedef {Object} Props
+ * @extends {Component<Props, Env>}
+ */
+export class BusLogsMenuItem extends Component {
+    static template = "bus.BusLogsMenuItem";
+    static components = { DropdownItem };
+
+    setup() {
+        this.busLogsService = useState(useService("bus.logs_service"));
+        this.downloadButton = useRef("downloadButton");
+    }
+
+    onClick(ev) {
+        if (this.downloadButton.el?.contains(ev.target)) {
+            this.env.services.bus_service.downloadLogs();
+        } else {
+            this.busLogsService.toggleLogging();
+            ev.preventDefault();
+            ev.stopPropagation();
+        }
+    }
+}
+
+registry
+    .category("debug")
+    .category("default")
+    .add("bus.download_logs", ({ env }) => ({
+        callback: () => env.services.bus_service.downloadLogs(),
+        Component: BusLogsMenuItem,
+        sequence: 1000,
+        section: "tools",
+        type: "component",
+    }));
```

Comment:
Better keep "Activate Test Mode" just before "Leave Debug Mode" at the bottom.
It doesn't feel right for "Enable Bus Logging" to be put in-between:

![Screenshot 2025-05-06 at 11 49 36](https://github.com/user-attachments/assets/d7597f22-925f-4cdd-997d-40d9a3ea9a10)

Would be best just above "Enable profiling" but the toggler items have too large height, so it looks bad. Another alternative is above "Become Superuser" with `sequence: 550` that looks better:

![Screenshot 2025-05-06 at 11 55 33](https://github.com/user-attachments/assets/50af2da8-88dc-4ef8-acd2-3cda0c7d4aae)